### ISSUE TITLE
Speed control should use acceleration and deceleration

### ISSUE THREAD

**Issue by KarlSchu** [SolarEnergyRacers/solar_car_control_system#138]
Recently only acceleration is used in speed control PID

-> on hold until battery is ready (for recuperation part)

**Comment by andreaskuster** [comment]
Idea:
Add a meta-variable for the PID controller `setpoint`

Pseudocode:
```
if (decelleration > 0){
   setpoint=  - deceleration 
} else { 
   setpoint = acceleration 
}
```

**Comment by andreaskuster** [comment]
@KarlSchu I assume this PR  #144 implements what you are looking for?

**Comment by andreaskuster** [comment]
Resolved with #173
